frontend/package.json

```diff
@@ -5,13 +5,15 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
-    "lint": "vue-cli-service lint"
+    "lint": "vue-cli-service lint",
+    "http:server": "http-server dist -c-1"
```

Comment:
Is it installed with another package? I believe it should be added to devDependencies.

Reply:
Good call. Adding it now.